azure-pipelines.yml

```diff
@@ -1,6 +1,11 @@
 # Runs the checks for Unified Target configurations
 #
 
+variables:
+  owner: betaflight
```

Comment:
What are `owner` and `repoName` used for?

Reply:
IDK, I was just following other repos' yml style. Shall we remove this?

Reply:
Yes please, thanks. Keeping unused variables around makes the code harder to understand and maintain.